lib/view_component/engine.rb

```diff
@@ -64,8 +64,8 @@ class Engine < Rails::Engine # :nodoc:
 
       if options.show_previews
         app.routes.prepend do
-          get "/rails/view_components"       => "rails/view_components#index", :internal => true
-          get "/rails/view_components/*path" => "rails/view_components#previews", :internal => true
+          get "/rails/view_components"       => "view_components#index", :internal => true
+          get "/rails/view_components/*path" => "view_components#previews", :internal => true
```

Comment:
I've often thought that we should make the mount point configurable but that is a separate PR.

Reply:
@juanmanuelramallo would you be willing to file this as an issue?

Reply:
https://github.com/github/view_component/issues/326

How does it look?

Reply:
@juanmanuelramallo looks good, thanks!